R/tutorial-format.R

```diff
@@ -24,6 +24,9 @@
 #'        \code{---} to em-dashes, \code{--} to en-dashes, and \code{...} to ellipses.
 #'        Deprecated in \pkg{rmarkdown} v2.2.0.
 #' @param ... Forward parameters to html_document
+#' @param language Language or custom text of the UI elements. See the
+#'        `multilang` vignette for more information about available options and
+#'        formatting.
```

Comment:
I still have trouble knowing where to look.

```suggestion
#' @param language Language or custom text of the UI elements. See 
#'        `vignette("multilang", package = "learnr")` for more information 
#'        about available options and formatting
```

Reply:
Does your suggestion fix the trouble or should we add more here?

Reply:
Sorry for being unclear. I have trouble remembering how to view a vignette. 

So we should use the suggestion above (or something similar), which gives me something to run or a link to click to view the information.